authui/src/authflowv2/base.css

```diff
@@ -51,6 +51,12 @@
     --border-radius-rounded-square: 0.875em;
   }
 
+  /* Border Weight */
+  :root {
+    --border-radius-regular: 1.5px;
```

Comment:
I think you mean `--border-width-regular: 1.5px`.

@tung2744 Any thoughts on `regular=1.5px & medium=2px`?

Reply:
Prefer calling `medium` as `active` or `highlight`, because the purpose of this border width is to indicate something is active / highlighted.

Reply:
`--border-width-regular: 1.5px`
`--border-width-regular--active: 2px`

Something like this?

Reply:
`--border-width-regular--active` this naming seems not so suitable to put in `base.css`? 
State related naming should put back to `component.css`?

Reply:
If the width is shared by multiple components, it can be placed in base.css